.github/workflows/playwright.yml

```diff
@@ -2,8 +2,6 @@ name: Automatisch UI Test
 on:
   push:
     branches: [main]
```

Comment:
Let's replace this with a scheduled run for the time being. We may set it up for once a day at noon.